CHANGELOG.md

```diff
@@ -51,6 +51,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Fix QueryPhaseResultConsumer incomplete callback loops ([#19231](https://github.com/opensearch-project/OpenSearch/pull/19231))
 - Fix the `scaled_float` precision issue ([#19188](https://github.com/opensearch-project/OpenSearch/pull/19188))
 - Fix Using an excessively large reindex slice can lead to a JVM OutOfMemoryError on coordinator.([#18964](https://github.com/opensearch-project/OpenSearch/pull/18964))
+- [Flakey Test] Fix flakey tests in SecureReactorNetty4HttpServerTransportTests ([#19327](https://github.com/opensearch-project/OpenSearch/pull/19327))
```

Comment:
I was going to complain about the use of "flakey" instead of the more common "flaky" but I'm not going to be that guy and instead say that we don't need a changelog entry for a test fix :)

Reply:
haha, I'll switch to the more conventional flaky for the future. [Merriam Webster](https://www.merriam-webster.com/dictionary/flaky) says the same.

> **flaky**, or less commonly **flakey**